P0009/P0009.md

```diff
@@ -2131,10 +2131,7 @@ a contiguous set of objects are accessed.
   * [2.2]{.pnum} a handle to a contiguous set of elements of type `element_type`,
     accessible through the policy's `access` method;
 
-  * [2.3]{.pnum} conversion of a handle to a contiguous set of elements,
-      to a pointer _[conv.array]_; and
-
-  * [2.4]{.pnum} getting a handle to the contiguous subset of elements
+  * [2.3]{.pnum} getting a handle to the contiguous subset of elements
```

Comment:
Should we take the opportunity to reword all these bullets?  It's a bit inconsistent how the other bullet points use "a handle to."  2.2 also refers to a "contiguous set of elements," but we don't promise contiguity.  Here's my attempt:

```
* [2.1]{.pnum} the type of a handle to a single element of type `element_type`;

* [2.2]{.pnum} a method to access the `i`-th element in a range of elements; and

* [2.3]{.pnum} a method to access the subrange beginning at an integer offset value of a given range of elements.
```
In any case, we do need "; and" at the end of 2.2 now, given that it's the penultimate point.

Reply:
yeah we should look at some of the other words to see if there is more stuff which needs to rewording. I agree that the previous wording was clunky. Do you want to issue a PR with your suggestion? Though 2.3 is not about access to a subrange, but more about creating a subrange.

Reply:
@crtrott I'll create a PR with these changes; thanks!

Reply:
PR with these changes and a few more: https://github.com/ORNL/cpp-proposals-pub/pull/171